scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/nnframes/NNClassifier.scala

```diff
@@ -318,10 +318,11 @@ object NNClassifierModel extends MLReadable[NNClassifierModel[_]] {
   }
 }
 
-class XGBClassifier () {
-  private val model = new XGBoostClassifier()
+class XGBClassifier (private[nnframes] val xgboostParams: Map[String, Any] = Map()) {
```

Comment:
Not sure if we need add `private[nnframes]` here, user maybe use bigdl 2.0 as a dependency and want to pass a xgboostParams in their examples which package name is not `nnframes`

Reply:
I have deleted `[nnframes]`.